dipy/segment/metric.py

```diff
@@ -1,15 +1,23 @@
 from dipy.segment.featurespeed import (Feature,
                                        IdentityFeature,
-                                       CenterOfMassFeature)
+                                       ResampleFeature,
+                                       CenterOfMassFeature,
+                                       MidpointFeature,
+                                       ArcLengthFeature,
+                                       VectorBetweenEndpointsFeature)
 
 from dipy.segment.metricspeed import (Metric,
                                       SumPointwiseEuclideanMetric,
                                       AveragePointwiseEuclideanMetric,
-                                      MinimumAverageDirectFlipMetric)
+                                      MinimumAverageDirectFlipMetric,
+                                      CosineMetric)
 
 from dipy.segment.metricspeed import (dist,
                                       distance_matrix)
 
+# Creates aliases
+EuclideanMetric = SumPointwiseEuclideanMetric
```

Comment:
@Garyfallidis Ok, I've been rethinking the framework. I'll make another PR explaining why but the gist of it is: what the hell I was thinking making `Metric` dependent on `Features`!

For now, I will remove the alias and modify the tutorial where it was used (i.e. the only place it was used anyway).